rest/resource-server/src/main/java/org/eclipse/sw360/rest/resourceserver/moderationrequest/Sw360ModerationRequestService.java

```diff
@@ -49,7 +54,7 @@ public class Sw360ModerationRequestService {
     private String thriftServerUrl;
 
     public static boolean isOpenModerationRequest(@NotNull ModerationRequest moderationRequest) {
-        return moderationRequest.getModerationState() == ModerationState.PENDING || moderationRequest.getModerationState() == ModerationState.INPROGRESS;
+        return moderationRequest.getModerationState() == ModerationState.PENDING || moderationRequest.getModerationState() == ModerationState.INPROGRESS || moderationRequest.getModerationState() == ModerationState.APPROVED;
```

Comment:
why you added approved state in isOpenModerationRequest method? please remove it